platform.md

```diff
@@ -477,9 +480,11 @@ Usage:
     - SHOULD set the modification time of all files in newly created layers to a constant value
     - SHOULD set the `created` time in image config to a constant value
 
-If a cache is provided the lifecycle:
-- SHALL write the contents of all cached layers to the cache
-- SHALL record the diffID and layer content metadata of all cached layers in the cache
+- The lifecycle SHALL write an [export report](#reporttoml-toml) to `<report>` describing the exported app image
+
+- *If* a cache is provided the lifecycle:
```

Comment:
Why do you use `*If*` instead of `**If**` like in other parts of this file?

Reply:
Oops! I'll fix it.